File: Basics/Conditional-Statements-Advanced/Exercise/volleyball.py
import math

year_type = input()
holiday_count = int(input())
hometown_weekend_count = int(input())
normal_weekend_count = 48 - hometown_weekend_count

normal_games = normal_weekend_count * 3/4
hometown_games = hometown_weekend_count
holiday_games = holiday_count * 2/3
total_games = normal_games + hometown_games + holiday_games

if year_type == "leap":
    total_games += total_games * 0.15

total_games = math.floor(total_games)

print(total_games)

File: JavaScript/Basics/0.3-Conditional-Steatements-Advanced/Exercise/volleyball.js
function volleyball(input) {
    let yearType = input[0];
    let holidaysCount = Number(input[1]);
    let weekendsCount = Number(input[2]);
    let gamesCount = (48 - weekendsCount) * (3 / 4);
    gamesCount += weekendsCount + (holidaysCount * (2 / 3));
    if (yearType == "leap") {
        gamesCount += gamesCount * 0.15;
    }
    gamesCount = Math.floor(gamesCount);
    console.log(gamesCount)
}

volleyball(["leap", "5", "2"])
volleyball(["normal", "3", "2"])
volleyball(["leap", "2", "3"])
volleyball(["normal", "11", "6"])
volleyball(["leap", "0", "1"])
volleyball(["normal", "6", "13"])